drivers/health_comprehensive_assessment/app/views/health_comprehensive_assessment/assessments/show.haml

```diff
@@ -13,7 +13,7 @@
     .mb-2
       %dt.inline Case Worker:
       %dd.inline= @assessment.user&.name
-  - if @client.patient.current_qa_factory.ca_completed_qa.present?
+  - if true # @client.patient.current_qa_factory.ca_completed_qa.present?
```

Comment:
Should there be a comment on the why this is always true?  Or maybe not an if?

Reply:
Debug code that crept it -- I just fixed it.

Reply:
ok, not seeing that change yet...

Reply:
Went straight to the staging branch -- I thought you were out today, so, I didn't wait to merge :-)